python/cucim/src/cucim/core/operations/morphology/_distance_transform.py

```diff
@@ -157,7 +157,7 @@ def distance_transform_edt(
         if len(unique_sampling) == 1:
             # In the isotropic case, can use the kernels without sample scaling
             # and just adjust the final distance accordingly.
-            scalar_sampling = float(unique_sampling)
+            scalar_sampling = float(unique_sampling[0])
```

Comment:
If this is a scalar, should we use `.item()`?

```suggestion
            scalar_sampling = float(unique_sampling.item())
```

Reply:
It is a 1d array of length 1, not an array scalar. I thought `item()` was only for array scalars, but I checked just now and it also works for arrays of size 1.

Reply:
just `float(unique_sampling)` will work for size 1 arrays.

Reply:
see [45fa113](https://github.com/rapidsai/cucim/pull/621/commits/45fa113c5689c1952bb5df34e66fd7a7d2bc9693)

Reply:
That's even better